index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+   
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,300;0,400;1,300&display=swap" rel="stylesheet">
+  
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <img src="" alt="Logo" id="header-img" >
+      <nav id="nav-bar">
+        <a href="#home" class="nav-link">Home</a>
+        <a href="#products" class="nav-link">Products </a>
+        <a href="#contact_us" class="nav-link">Contact us</a>
+      </nav>
+    </header>
+    <div id="home">
+      <!-- <video width="600" height="240" src="https://www.youtube.com/watch?v=ScMzIvxBSi4" autoplay controls preload="none"muted>
+        Video cannot be displayed
+      </video> -->
```

Comment:
I kindly suggest that you remove unused code, as it will make your code bulky and difficult to maintain.